src/components/ResourceList/ResourceList.tsx

```diff
@@ -90,6 +90,8 @@ export interface ResourceListProps {
   idForItem?(item: any, index: number): string;
   /** Function to resolve an id from a item */
   resolveItemId?(item: any): string;
+  /** React node to display instead of the default empty state */
+  alternateEmptyState?: React.ReactNode;
```

Comment:
alternative to what? the default? That seems kinda implied already :)  Just `emptyState` perhaps?

Reply:
If you look above, the other customizable `ReactNode` prop in `ResourceList` is called `alternateTool`, so I thought I'd use the same format

Reply:
Empty state isn't a concept that exists in resource list yet (I added it because that's where we're heading with the new design lang, but I had to revert and haven't had a free cycle to ship a revert of the revert). We could call it `emptySearchResultMarkup` since that's the component the markup set on this prop replaces?